TUTORIAL.md

```diff
@@ -159,6 +159,40 @@ If you encounter 'Connection reset by peer' errors when cloning, run, say, `assi
 (Assigner doesn't have plans for any grading features;
 however, if you are interested in automated grading, check out its sister project, [grader](https://github.com/redkyn/grader).)
 
+### Uploading scores to Canvas
+
+Although Assigner does not support grading, it can be used to upload the scores generated by an automated grader to Canvas.
+Currently, this feature is supported for workflows in which an automated grading script runs as part of a Continuous Integration (CI) job and 
+produces an output file (artifact) containing the score.
```

Comment:
We need some documentation on how to set up these CI jobs and links to an example CI script repository.

Reply:
https://github.com/grade-sh has complete functional C++ and Python examples.

Reply:
Cool! I wasn't sure the status of those since I'd looked last. Seems like they're still private, though. I know they aren't my projects, but I would like to request a little bit of documentation for them so people know where to start modifying things for their assignments. I want to avoid a situation where someone says, "oh cool, I can grade things automatically now?", opens the example, thinks "ah, I'll figure out how to make this work later," and then 'later'—well, you know how 'later' works in academia :)

Reply:
Thanks to Josh, we've got much better documentation, along with the CI actually running the examples in Gitlab, public now: https://gitlab.com/classroomcode/py2cfg

Reply:
Fantastic!

Reply:
https://gitlab.com/classroomcode/grade-sh/grade-sh